pages/en/new-design/index.mdx

```diff
@@ -16,50 +16,79 @@ layout: home.hbs
   <div>
     <WithNodeRelease status={['Active LTS', 'Maintenance LTS']}>
       {({ release }) => (
-        <DownloadButton release={release}>Download Node.js (LTS)</DownloadButton>
```

Comment:
It's a cool change that's been made. But I don't think it's a good idea to have the same two buttons. What Hayden has done in the design is more intelligent, putting two buttons with different styles creates an asymmetry that highlights one. I'm not an expert in design, I'm just learning. I think asking Hayden's opinion wouldn't be a bad idea.

Reply:
Right, and I agree with you. These are experimental changes, because after all who approves these designs and content changes is the TSC, not Hayden.

So as much as I value his opinions and vision here, in the end what the TSC decides is what should be done.

There are still ongoing discussions regarding if we should have two buttons or not and there is no clear consensus yet.